charts/artifact-hub/templates/hub_service.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: hub
+  name: {{ include "chart.resourceNamePrefix" . }}hub
```

Comment:
The [service name used in the hub ingress in the default backend](https://github.com/artifacthub/hub/blob/e27733fc29329abd32b81eff48591387176bc54d/charts/artifact-hub/templates/hub_ingress.yaml#L19) as well as in other backends referenced from rules should probably use the prefix too. The default backend service name is hardcoded in the ingress template at the moment, and in the [staging](https://github.com/artifacthub/hub/blob/e27733fc29329abd32b81eff48591387176bc54d/charts/artifact-hub/values-staging.yaml#L25) and [production](https://github.com/artifacthub/hub/blob/e27733fc29329abd32b81eff48591387176bc54d/charts/artifact-hub/values-production.yaml#L33) values files you can see it used in the rules. We also use the service name in the [readinessGates configuration](https://github.com/artifacthub/hub/blob/e27733fc29329abd32b81eff48591387176bc54d/charts/artifact-hub/values-production.yaml#L36-L37) in staging and production values files.

Reply:
Thanks for the hint! As mentioned in a comment above, I'm using a route instead of the Chart's ingress. I'll fix it in the next commit.
Concerning the values: Are those considered best practice staging and production environments, or are they just used for the official ArtifactHub staging and production environment? Technically those values are still correct, since by default the dynamic resource naming is disabled.

Reply:
Awesome, thanks 👍 

They are just the ones used for the official staging and production environments (except for credentials, etc). Yes, nothing will break immediately, but we may reinstall any or both of the environments soon again and that might change. It'll be better if they are ready to work with the prefix if we go that way eventually, not keeping any hardcoded values.

Reply:
Unfortunately both, the ingress rules and the readiness gates highly depend on the cluster.
Therefore I don't really see an option to create those values dynamically. They need to be adapted if ArtifactHub is reinstalled and the dynamic resource naming is enabled.
I could add a comment though...
Or do you see any other options?

Reply:
I fixed the ingress' service reference in the last commit.
So the only thing in this PR which remains in my opinion is how to handle the defaults for the ingress and  the readiness gates in the staging and production values file.

Reply:
> Or do you see any other options?

I can't think of anything either. But as you mentioned these values are specific for a particular cluster, so I think it's fair enough to adapt them if something changes. A comment is a great idea though.

Reply:
I added the comments to both `values-staging.yaml` and `values-production.yaml`.
The CI jobs unfortunately failed in the setup stage with the following error:
```
Error: An action could not be found at the URI 'https://api.github.com/repos/actions/checkout/tarball/5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f'
```
Maybe you could restart the pipeline?

Reply:
Done! 👍